ouroboros-consensus/src/Ouroboros/Consensus/Storage/ImmutableDB/Chunks/Layout.hs

```diff
@@ -51,14 +51,10 @@ import           NoThunks.Class (NoThunks)
 import           Ouroboros.Consensus.Block
 
 import           Ouroboros.Consensus.Storage.ImmutableDB.API (Tip (..))
+import           Ouroboros.Consensus.Storage.ImmutableDB.Chunks.Internal
 import           Ouroboros.Consensus.Storage.ImmutableDB.Impl.Types
                      (BlockOrEBB (..))
 
--- Most types in the Chunks interface are opaque in the public API, since their
--- interpretation is subject to layout decisions. In this module we /make/ those
--- layout decisions, however, and so here we need access to the internal types.
```

Comment:
Does this suffice @amesgen @Jasagredo ? Doesn't look like there are that many files in our codebase that contain comments between the first and last `import` statements.

```
ouroboros-network$ cat quick-script.sh
find *consensus* -type f -name '*hs' -print0 | while read -d $'\0' file
do
    i=$(grep -m1 -n -E -e '^import' "$file"            | cut -f1 -d:)
    j=$(grep     -n -E -e '^import' "$file" | tail -n1 | cut -f1 -d:)
    test -n "$i" && {
        n=$(expr "$j" - "$i" + 1)
	comments=$(tail -n+"$i" "$file" | head -n"$n" | grep -E -e '^ *--' -e '^ *{-')
	test -n "$comments" && { echo "$file $i $j"; echo "$comments"; }
    }	 
done

ouroboros-network$ . quick-script.sh
ouroboros-consensus/src/Ouroboros/Consensus/HeaderStateHistory.hs 25 44
-- Support for tests
ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/API.hs 67 100
-- Support for tests
ouroboros-consensus/src/Ouroboros/Consensus/Storage/ImmutableDB/Impl/Types.hs 19 30
-- Importing from Internal to avoid circular dependency
ouroboros-consensus/src/Ouroboros/Consensus/Storage/ImmutableDB/Chunks.hs 4 7
-- Only export public API from the Internal module
ouroboros-consensus/src/Ouroboros/Consensus/Storage/ImmutableDB/Chunks/Layout.hs 46 60
-- Most types in the Chunks interface are opaque in the public API, since their
-- interpretation is subject to layout decisions. In this module we /make/ those
-- layout decisions, however, and so here we need access to the internal types.
ouroboros-consensus/src/Ouroboros/Consensus/Util/MonadSTM/NormalForm.hs 15 36
-- TODO: use strict versions of 'TQueue' and 'TBQueue'.  Previously the
-- 'Control.Monad.Class.MonadSTM.Strict' was imported which
-- exported lazy 'TQueue' and 'TBQueue',  I (@coot) think that the intention was
-- to use strict versions.
ouroboros-consensus/src/Ouroboros/Consensus/Util/Condense.hs 13 49
-- Imports from lower-level libs to avoid circular dependencies
ouroboros-consensus/src/Ouroboros/Consensus/Util/CBOR.hs 32 64
--import           Ouroboros.Consensus.Block
ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator.hs 9 98
-- Defines 'SingleEraBlock' and 'CanHardFork'
-- Defines 'HardForkProtocol', 'HardForkBlock', and 'LedgerState',
-- as well as various config types.
-- Defines 'HasPartialConsensusConfig' and 'HasPartialLedgerConfig'
-- Instances for 'BlockHasCodecConfig', 'GetHeader', 'HasHeader',
-- 'BasicEnvelopeValidation'
-- Instances for 'IsLedger', 'ApplyBlock', 'UpdateLedger',
-- 'LedgerSupportsProtocol', 'HasHardForkHistory', 'ValidateEnvelope'
-- Instances for 'ApplyTx', 'HasTxId', 'HasTxs'
-- Instance for 'ConsensusProtocol'
-- Instance for 'CommonProtocolParams'
-- Instance for 'LedgerSupportsPeerSelection'
-- Instances for 'ShowQuery' and 'QueryLedger'
-- Definition of 'Query', required for serialisation code
-- Instance for 'ChainSelection'
-- Re-export only the types that are useful in RunNode instances for HFC blocks
-- Re-export types required to initialize 'ProtocolInfo'
-- Defines the various translation types required for concrete HFC instances
-- Combinator for 'BlockForging'
-- Instances for 'RunNode' and 'ConfigSupportsNode'
-- Instance for 'NodeInitStorage'
-- Instance for 'BlockSupportsMetrics'
-- Definition of InPairs (required to define translations)
-- Definition of Telescope (required to define serialisation code)
-- Definition of 'Mismatch' (required to define serialisation code)
-- Definition of HardForkState (required to define serialisation code)
-- Also export functions required to define 'protocolInfo'.
-- Definition of 'InjectTx' (required to define tx injections)
ouroboros-consensus/src-unix/Ouroboros/Consensus/Storage/IO.hs 16 32
-- Package 'unix' exports the same module.
ouroboros-consensus/src-win32/Ouroboros/Consensus/Storage/IO.hs 15 31
-- Dodgy import when Win32 >= 2.6.2.0
ouroboros-consensus-test/test-storage/Test/Ouroboros/Storage/LedgerDB/OnDisk.hs 45 96
-- For the Arbitrary instance of 'MemPolicy'
```

Reply:
I relocated/removed most of those comments. Most of them were outdated or irrelevant I think.